shielder/contract/drink_tests/mod.rs

```diff
@@ -0,0 +1,226 @@
+mod utils;
+
+use drink::{
+    runtime::MinimalRuntime,
+    session::Session,
+    AccountId32,
+};
+
+use crate::{
+    contract::OpPub,
+    drink_tests::utils::{psp22::*, shielder::*},
+    mocked_zk::{ops::OpPriv, tests::merkle::MerkleTree},
+    types::Scalar,
+};
+
+#[drink::contract_bundle_provider]
+pub enum BundleProvider {}
+
+#[drink::test]
+fn deploy_single_deposit_single_withdraw() -> Result<(), Box<dyn std::error::Error>> {
+    const ALICE: AccountId32 = AccountId32::new([1u8; 32]);
+    const BOB: AccountId32 = AccountId32::new([2u8; 32]);
```

Comment:
this can be moved outside, so that they can be reused in other testcases

Reply:
moved to init method